keylime/registrar_client.py

```diff
@@ -75,8 +83,8 @@ def getData(registrar_ip, registrar_port, agent_id):
 
     response = None
     try:
-        client = RequestsClient(f'{registrar_ip}:{registrar_port}', tls_enabled)
-        response = client.get(f'/v{api_version}/agents/{agent_id}', cert=tls_cert_info, verify=False)
+        client = RequestsClient(f'{registrar_ip}:{registrar_port}', tls_enabled, ignore_hostname=True)
```

Comment:
I think that now most of the `RequestsClient` calls makes `ignore_hostname` to `True`, except for tenant_webapp.

Do you think that can be better to remove it as a default parameter, and make it explicit?

Reply:
We could, but I like that you have to explicitly write that hostnames are ignored and are otherwise more secure by default.

Disabling the hostname validation in TLS is a trade off with we currently deploy certificates and security. I think that in the future we can remove most of the `ignore_hostname=True` if certificates contain the correct SAN attributes.

Reply:
I see, thanks!